samples/nrf_desktop/src/hw_interface/Kconfig

```diff
@@ -63,6 +63,39 @@ config DESKTOP_OPTICAL_DEFAULT_CPI
 	  This value is set on firmware start. It can be changed in runtime
 	  using the configuration channel.
 
+config DESKTOP_OPTICAL_RUN_DOWNSHIFT_TIME_MS
+	int "Default run downshift time of the mouse optical sensor"
+	depends on DESKTOP_MOTION_OPTICAL_ENABLE
+	default 500
+	range 10 2550
+	help
+	  Time after which sensor goes from "Run" to "Rest 1" mode.
+
```

Comment:
excessive blank line
also in other options

Reply:
done